tests/aws/services/kinesis/test_kinesis.py

```diff
@@ -413,6 +413,24 @@ def test_get_records_next_shard_iterator(
         assert shard_iterator != get_records_response["NextShardIterator"]
         assert new_shard_iterator != get_records_response["NextShardIterator"]
 
+    @markers.aws.validated
+    def test_get_records_shard_iterator_with_surrounding_quotes(
+        self, kinesis_create_stream, wait_for_stream_ready, aws_client
+    ):
+        stream_name = kinesis_create_stream(ShardCount=1)
+        wait_for_stream_ready(stream_name)
+
+        first_stream_shard_data = aws_client.kinesis.describe_stream(StreamName=stream_name)[
+            "StreamDescription"
+        ]["Shards"][0]
+        shard_id = first_stream_shard_data["ShardId"]
+
+        shard_iterator = aws_client.kinesis.get_shard_iterator(
+            StreamName=stream_name, ShardIteratorType="LATEST", ShardId=shard_id
+        )["ShardIterator"]
+
+        aws_client.kinesis.get_records(ShardIterator=f'"{shard_iterator}"')
```

Comment:
Is there a scenario where there might be single quotes? Do we need to cover both cases?

Reply:
I did not check whether single quotes would be allowed, but the main issue of the customer was that the aws cli returns it wrapped in double quotes, so this is the most important case for now :)